src/routes/Login/index.tsx

```diff
@@ -1,14 +1,114 @@
-import React from 'react';
-import { useLoginHook } from './hooks';
-import SharedButton from '../../components/shared/SharedButton';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { SIGNUP_ROUTE } from './../../utils/routesConstants';
+import './Login.scss';
```

Comment:
@alokesh985 Change this file name to `./login.scss`.

Reply:
Done.